### ISSUE TITLE
TV Multi EPG and times

### ISSUE THREAD

**Issue by ims21** [E2OpenPlugins/e2openplugin-OpenWebif#258]
As I saw tonight on cca 01:30, all epg on TV Multi EPG begin on 4:00 and more. Nothing between 01:30 - 4:00. On all services and all bouquets. Why ? Channel EPG is ok.

**Comment by jbleyel** [comment]
strange

**Comment by ims21** [comment]
may be, but now again...

**Comment by Priskvark** [comment]
Yes, the same thing happens on my Vu+ Solo2.

The new clock-day starts after midnight but the new “TV-day”  starts at 06:00. Maybe there has to be an adjustment so that OWIF uses TV-day calendar rather than clock a clock day calendar in the EPG-window? 

This explanation sounds a bit confusing at first. I know ;-)

**Comment by ims21** [comment]
yes, it could be ...when is selected next day, then EPG begin cca around 5-6h am. 
May be, when is time over > 24h and < "daybreak", is displayed nothing due some bug...
